.github/workflows/ci.yaml

```diff
@@ -19,8 +19,8 @@ jobs:
       fail-fast: false
       matrix:
         env:
-          - IMAGE: rolling-ci
-            ROS_DISTRO: rolling
+          - IMAGE: humble-ci
+            ROS_DISTRO: humble
```

Comment:
As noted, industrial CI tries to install mavros via`apt`, which is currently not available for rolling (#248).  

CI _does_ work if we force it to use Humble, for which there's a binary image. 

Note this is _just_ the ros_industrial CI stage, not building the Docker images (where Mavros is built from source for rolling/jazzy.

Reply:
Thanks for looking into that. Because we know it's an issue with MAVROS, It's not a big deal if the CI fails in this PR. We can just update the CI stage in #241 to use the new `mavros` stage.